federation-js/src/composition/__tests__/compose.test.ts

```diff
@@ -489,8 +531,34 @@ describe('composeServices', () => {
         expect(schema).toBeDefined();
         expect(errors).toMatchInlineSnapshot(`
           Array [
-            [GraphQLError: Field "Product.name" can only be defined once.],
-            [GraphQLError: There can be only one type named "Product".],
+            Object {
+              "code": "MISSING_ERROR",
+              "locations": Array [
+                Object {
+                  "column": 3,
+                  "line": 4,
+                },
+                Object {
+                  "column": 3,
+                  "line": 4,
+                },
+              ],
+              "message": "Field \\"Product.name\\" can only be defined once.",
+            },
+            Object {
+              "code": "MISSING_ERROR",
+              "locations": Array [
+                Object {
+                  "column": 1,
+                  "line": 2,
+                },
+                Object {
+                  "column": 6,
+                  "line": 2,
+                },
```

Comment:
I would expect these both to either point at the "t" in type or the "P" in Product. It's entirely plausible this is out of our hands since this error is coming from `graphql-js`.

Reply:
line 2, column 1 is the t in type & line 2, column 6 is the P in Product, as far as I can tell.

Reply:
Right, I would expect both locations to point to the same thing, i.e. both pointing to the "t", or both pointing to the "P"

Reply:
oh duh 🤦 thank you for the clarification. That is weird, I will look into where these errors came from. Why is there two in the first place?

Reply:
There are two places where type `Product` is defined, so it's pointing to both of them